jsonnet/components/prometheus.libsonnet

```diff
@@ -366,7 +366,7 @@ function(params)
               '-upstream=http://localhost:9090',
               '-openshift-service-account=prometheus-k8s',
               '-openshift-sar={"resource": "namespaces", "verb": "get"}',
-              '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get"}}',
+              '-openshift-delegate-urls={"/api": {"resource": "namespaces", "verb": "get"},"/metrics": {"resource": "namespaces", "verb": "get"},"/federate": {"resource": "namespaces", "verb": "get"}}',
```

Comment:
i'm not sure that we need to declare `/metrics` since the metrics should be scraped from the kube-rbac-proxy container?

Reply:
I went by what the functional tests look for. https://github.com/openshift/origin/blob/master/test/extended/prometheus/prometheus.go#L546 checks for `/metrics` on port 9091.

Reply:
Ok, can you add a TODO comment to be fixed once 4.11 opens?